pos_default_quantity/models/product.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# © 2016 Robin Keunen, Coop IT Easy SCRL fs
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from openerp import models, fields, api
+
+
+class ProductUomCategory(models.Model):
+    _inherit = 'product.uom.categ'
+
+    pos_default_qty = fields.Float(
```

Comment:
Why not to use something like a default_pos_uom_id and take the conversion from its conversion factor?

Reply:
I am not sure I understand your suggestion, the goal of this PR is to allow setting a default quantity when initialising an orderline in the pos. The use case being : 
- if product uom is unit, default qty is 1
- if product uom is weight, default qty is 0

Thanks for the review btw

Reply:
My above comment is misleading. The use case is :

- if product uom **category** is unit, default qty is 1
- if product uom **category** is weight, default qty is 0